packages/ui/src/components/templates/report/components/WebsiteCredibility/WebsiteCredibility.tsx

```diff
@@ -12,107 +50,229 @@ export const WebsiteCredibility: FunctionComponent<{
-                    </span>
-                  )}
-                </li>
-              ))}
-            {!onlineReputationAnalysis?.length && <li>No Indications Detected.</li>}
-          </ol>
-        </CardContent>
-      </Card>
-      <Card>
-        <CardHeader className={'pt-4 font-bold'}>Pricing Analysis</CardHeader>
-        <CardContent>
-          <ol
-            className={ctw({
-              'ps-4': !!pricingAnalysis?.length,
-            })}
-          >
-            {!!pricingAnalysis?.length &&
-              pricingAnalysis.map(warning => (
-                <li key={warning} className={'list-decimal'}>
-                  {warning}
-                </li>
-              ))}
-            {!pricingAnalysis?.length && <li>No Indications Detected.</li>}
-          </ol>
-        </CardContent>
-      </Card>
-      <Card>
-        <CardHeader className={'pt-4 font-bold'}>
-          Website Structure and Content Evaluation
-        </CardHeader>
-        <CardContent>
-          <ol
-            className={ctw({
-              'ps-4': !!websiteStructureAndContentEvaluation?.length,
-            })}
-          >
-            {!!websiteStructureAndContentEvaluation?.length &&
-              websiteStructureAndContentEvaluation.map(warning => (
-                <li key={warning} className={'list-decimal'}>
-                  {warning}
-                </li>
-              ))}
-            {!websiteStructureAndContentEvaluation?.length && <li>No Indications Detected.</li>}
-          </ol>
+    <h3 className="pt-4 font-bold">Traffic Analysis</h3>
+
+    {/* <div className="flex flex-col 2xl:flex-row gap-4 w-full h-auto 2xl:h-96">
+      <div className="h-[24rem] 2xl:h-full w-full 2xl:w-3/5"> */}
+    <div className="flex gap-4 h-[30rem] w-full">
+      <Card className="h-full w-3/5">
+        <CardHeader className="pt-4 font-bold">Estimated Monthly Visitors</CardHeader>
+
+        <CardContent className="h-4/5 w-full pb-0 mt-auto">
+          <ResponsiveContainer width="100%" height="100%">
+            <BarChart
+              data={trafficAnalysis.montlyVisitsIndicators}
+              margin={{ top: 5, right: 30, left: 20, bottom: 5 }}
+              barSize={46}
+            >
+              <CartesianGrid vertical={false} strokeDasharray="0" />
+              <XAxis
+                dataKey="label"
+                tickFormatter={value => dayjs(value, 'MMMM YYYY').format('MMM YYYY')}
+              />
+              <YAxis tickFormatter={Intl.NumberFormat('en', { notation: 'compact' }).format} />
+              <RechartsTooltip
+                content={({ active, payload, label }) => {
+                  if (active && payload && payload.length) {
+                    return (
+                      <div className="bg-white border border-gray-400 rounded-md px-4 py-2 text-gray-600">
+                        <p className="max-w-xs">{`On ${label} the company's website had approx. ${
+                          payload.at(0)?.value
```

Comment:
Formatting the number will make it easier to read.

```suggestion
                          new Intl.NumberFormat().format(payload.at(0)?.value)
```